x-pack/test/upgrade/apps/dashboard/dashboard_smoke_tests.ts

```diff
@@ -41,7 +42,9 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         dashboardTests.forEach(({ name, numPanels }) => {
           it('should launch sample ' + name + ' data set dashboard', async () => {
             await PageObjects.home.launchSampleDashboard(name);
-            await PageObjects.timePicker.setCommonlyUsedTime('Last_1 year');
+            await kibanaServer.uiSettings.update({
+              'timepicker:timeDefaults': `{ "from": "now-5y", "to": "now"}`,
+            });
```

Comment:
I'm not sure but you probably need to refresh page/ navigate to dashboard again so the changes will take place. Does it work locally?